### PR TITLE
Add helm chart prom-rds-exporter

### PR DESCRIPTION
**Pull request by dungpham91**
Hi,

- I created a Helm chart for rds_exporter and named it prom-rds-exporter.
- I tested in my EKS cluster and it's working fine.
- Also created a README for this chart, by it, you can install it easily.

If you guys can take few mins to review and approve it. I think it's will be more convenience for the other guys when using it.

Thanks.

PS: my old PR https://github.com/percona/rds_exporter/pull/89

### CODE REVIEW

**Comment by jagadeeshmedavarapu1** [comment]
Can we define cross-account role for RDS-Exporter..? Is it supportable?

**Comment by dungpham91** [comment]
> Can we define cross-account role for RDS-Exporter..? Is it supportable?

I only have one account so I've never tried this. You can try it with cross-account role. In theory, the role for the exporter only needs to ensure the policies as in the README I wrote.

**Review by BupycHuk — DISMISSED** [review]
Looks good from rds_exporter configuration side. @tplavcic can someone from Cloud Native team review from K8s perspective?

**Review by tplavcic — DISMISSED** [review]

File: charts/prom-rds-exporter/values.yaml [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+    - host: chart-example.local
+      paths:
+        - path: /
+          pathType: ImplementationSpecific
+  tls: []
+  #  - secretName: chart-example-tls
+  #    hosts:
+  #      - chart-example.local
+
+securityContext:
```
Comment (egegunes) [inline charts/prom-rds-exporter/values.yaml:222]: securityContext is duplicated

Comment (dungpham91) [reply charts/prom-rds-exporter/values.yaml:214]: Hi @egegunes 
Thanks for comment. I've updated the chart and also check `helm lint` from my locally. Please verify again. Thanks.

**Pull request closed without merging**